helm/opendistro-es/files/configurer/kubeaudit_template.json

```
@@ -0,0 +1,11 @@
+{
+    "index_patterns": [
+      "kubeaudit*"
+    ],
+    "settings": {
+      "index.refresh_interval": "10s",
+      "opendistro.index_state_management.policy_id": "kubeaudit",
+      "opendistro.index_state_management.rollover_alias": "kubeaudit",
+      "mapping.total_fields.limit": "1000"
+    }
+  }
```

I see the the indentation is off for all json files. Maybe we should only use an indentation of 2 spaces?
```suggestion
{
  "index_patterns": [
    "kubeaudit*"
  ],
  "settings": {
    "index.refresh_interval": "10s",
    "opendistro.index_state_management.policy_id": "kubeaudit",
    "opendistro.index_state_management.rollover_alias": "kubeaudit",
    "mapping.total_fields.limit": "1000"
  }
}
```

---

Fixed :slightly_smiling_face: